### ISSUE TITLE
version 0.3.15 and 0.3.16 are not returning compare data

### ISSUE THREAD

**Issue by spandl** [googledatastudio/tooling#140]
This issue has also been posted here:
https://issuetracker.google.com/issues/152250227

But I think opening an issue here is more appropriate.

Created a basic community visualization and configured the chart to return a compare data range.
This returns usually values in the DEFAULT table as well as the COMPARISON table

```
tables: {DEFAULT: Array(1), COMPARISON: Array(1)}
```

It is totally working when the package is using version 0.3.13

```
"@google/dscc": {
            "version": "0.3.13",
            "resolved": "https://registry.npmjs.org/@google/dscc/-/dscc-0.3.13.tgz",
            "integrity": "sha512-QEs3kJVOvtx4rANe5US217VBaC1k9P1N+9zb60D1twdjJvdjP5JLY5bD2XRbCOT31HeFL+wTl2ZSmjfH1C4dgQ==",
            "requires": {
                "url-parse": "^1.4.7"
            }
        },
```

It is not working if the installation is using version 0.3.15 or 0.3.16 (x.14 was not tested)

```
"@google/dscc": {
            "version": "0.3.15",
            "resolved": "https://registry.npmjs.org/@google/dscc/-/dscc-0.3.15.tgz",
            "integrity": "sha512-GOf+t99cGbT3U9j8cfYDXMVNkvl5HZdu23J5FhsADrCPOqW2E3i7/8RsGT1eEWqbC+FVBBRb3R/7Cr/+k/DTcw==",
            "requires": {
                "url-parse": "^1.4.7"
            }
        },
```

When using these versions, the COMPARISON data ARRAY is always empty ([])

***
**What you expected to happen:**
COMPARISON data ARRAY should contain values

**Steps to reproduce:**
Update any CV chart which is using compare data with one of the packages compile and configure with compare data. The table is always empty.

**Other information (workarounds you have tried, documentation consulted, etc):**
The only way to make this work is re-installing the repo with package 0.3.13 and everything works as expected.

**Comment by spandl** [comment]
Maybe it happened here:

**Update**
https://github.com/googledatastudio/tooling/commit/fa7f9d7e3a36b4c4e8d8b22be5d61ed687a4442c#diff-899026fcb489680427fd1853dd744bd8

**File:**
packages/ds-component/src/index.ts

**Comment by spandl** [comment]
I can confirm that the fix in version 0.3.17 fixes the issue. 
Thanks @y3l2n !
